src/components/table/index.tsx

```diff
@@ -321,6 +321,7 @@ export default function EnhancedTable<T>({
                                 color='primary'
                                 checked={isItemSelected}
                                 onClick={(e) => (!isClickable || !isClickable(row as T) ? handleClick(e, row as T) : null)}
+                                {...CheckboxIcons}
```

Comment:
Could we replace the MUI checkbox with the web-component checkbox here so we don't need to pass the icons?

Reply:
This was not working well for some reason. I will try again.

Reply:
Our checkbox is a labeled checkbox.. which has different semantics and event handling.
For now I'm going to keep them this way. We can do a refactor later. It cannot be reused the way it is now.

Reply:
Thanks for trying!